musit-models/src/main/scala/no/uio/musit/functional/MonadTransformers.scala

```diff
@@ -53,6 +55,25 @@ object MonadTransformers {
 
   }
 
+  object MusitResultT {
+
+    /**
+     * Returns a MusitResultT[Future, A] where the Future is successfully
+     * evaluated with the given MusitResult[A] argument.
+     *
+     * @param res the MusitResult to wrap
+     * @param m an implicitly provided Monad[Future]
+     * @tparam A the type we want to access inside the transformer functions
+     * @return {{{MusitResultT[Future, A]}}}
+     */
+    def successful[A](
```

Comment:
Yeah, if we're taking that path we have to find the balance so it's not to much black magic for the less experiences developers.